package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@adobe/aio-lib-cloudmanager",
-  "version": "2.0.0",
+  "version": "3.0.0",
```

Comment:
If I'm not mistaken, this needs to be handled by `semantic-release` bot.

Considering https://github.com/adobe/aio-lib-core-networking/issues/16 is a breaking change, I agree we should have this update in a major version, but I've looked through https://github.com/adobe/aio-lib-cloudmanager/blob/main/.github/CONTRIBUTING.md#commits-and-releasing and it seems we need to append `BREAKING CHANGE: <description>` into our commits for the `semantic-release-bot` to create a `3.0.0`.

Here's an example that I've found: #359.

Reply:
Ok, thanks. Let me revert that change and add a BREAKING CHANGE line to the commit